.github/workflows/run_tests.yml

```diff
@@ -17,16 +17,31 @@ jobs:
     - name: checkout git repo
       uses: actions/checkout@v3
 
-    - name: pull sdk image and test
-      shell: bash
-      env:
-        KBASE_TEST_TOKEN: ${{ secrets.KBASE_TEST_TOKEN }}
-        GITHUB_ACTIONS_CI: 1
+    - uses: actions/setup-python@v4
```

Comment:
the tests don't require the SDK or any external connections, so I changed the set up so they run on the standard GitHub actions ubuntu image and send the code coverage file to Codecov.